test/tools/jsontests/vm.cpp

```diff
@@ -296,8 +296,9 @@ namespace dev { namespace test {
 
 json_spirit::mValue doVMTests(json_spirit::mValue const& _input, bool _fillin)
 {
-	if (string(boost::unit_test::framework::current_test_case().p_name) != "vmRandom")
-		TestOutputHelper::initTest(_input.get_obj().size());
+	unique_ptr<TestOutputHelper> testOutputHelper;
+	if (string(boost::unit_test::framework::current_test_case().p_name) != "vmRandom") // TODO: remove this hard-coding
+		testOutputHelper.reset(new TestOutputHelper(_input.get_obj().size()));
```

Comment:
All those `TestOutputHelper const*` arguments appeared when I tried to remove this hard-coded "vmRandom", but I now leave this.

Reply:
Maybe to get rid of it we could make `doVMTests` and separate `doVMTestsWithOutput` that creates `TestOutputHelper` then calls `doVMTests`

Then call internal `doVMTests` only from `vmRandom`

Reply:
I see.  I'll name the new function `doVMTestsNoLog()` following the similar function `doBlockchainTestsNoLog()`.